delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/disk/package.scala

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations
+
+import akka.http.scaladsl.model.Uri.Path
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.files.model.FileAttributes
+
+import java.net.URI
+import java.nio.file
+import java.nio.file.Paths
+
+package object disk {
+
+  def absoluteDiskPathFromAttributes(attr: FileAttributes): IO[file.Path] = absoluteDiskPath(attr.location.path)
+
+  def absoluteDiskPath(relative: Path): IO[file.Path] = IO(Paths.get(URI.create(s"file://$relative")))
```

Comment:
My instinct here is that stuff touching the file system would be better off in a class so it's mockable, perhaps that doesn't apply here

Reply:
The fs2 `Files[IO]` probably is this abstraction (or maybe a smaller wrapper round it). This was just removing duplication again